### ISSUE TITLE
Convert a data matrix to Image type and display it via view do not work

### ISSUE THREAD

**Issue by BoundaryValueProblems** [JuliaImages/Images.jl#545]
I'm simply trying to display a data matrix of type float64 using the ImageView.view function.
This particular matrix ranges from 23.0 to 226.0, so it's already within 0 to 255. Hence, I thought that the following would work; say, data below is a float 64 array of size 128 x 128:

```
julia> img=convert(Image, round(UInt8, data))
Gray Images.Image with:
  data: 128x128 Array{UInt8,2}
  properties:
    timedim: 0
    colorspace: Gray
    colordim: 0
    spatialorder:  y x
    pixelspacing:  1.0 1.0
julia> view(img,pixelspacing=[1,1])
```

This pops up an ImageView window, but nothing is displayed inside the window; it's simply empty. I would appreciate it if you could point me out what is wrong here.
Thanks a lot!

**Comment by timholy** [comment]
Intensities always default to the range 0->1. See the Images docs.

**Comment by BoundaryValueProblems** [comment]
Thank you very much! It worked after scaling it within [0,1] range.
It would be great if you could put a link to that Images docs in README.md since there may be many people who want to display a given matrix that is **not** loaded from typical image format files such as .png/.jpg files as a grayscale image using the "view" function. I think the link to the Images docs is the following, am I right?
https://github.com/timholy/Images.jl/blob/master/docs/src/core.md
Thanks!

**Comment by timholy** [comment]
Already described at https://github.com/timholy/Images.jl#loading-your-first-image-how-images-are-represented?

**Comment by BoundaryValueProblems** [comment]
Thanks, Tim!
I missed that portion when I read README.md.
Perhaps, it may be a good idea to emphasize that paragraph.

**Comment by timholy** [comment]
Good to know. I am rewriting Images and its documentation now, so that's timely advice.

**Comment by BoundaryValueProblems** [comment]
Thanks, Tim.
Something like the following example will help many people (I used your earlier posting as an example).
```
# viewing some synthetic matrix data of size 500 x 500 using grayscale
julia> y = linspace(0,2pi,500); x=y';data=sin(x .+ cos(x .* y))
julia> view(convert(Image, map(ScaleAutoMinMax(U8),data)))
```
Thanks!

**Comment by timholy** [comment]
Closed by #577
